reading_order_widget.py

```
@@ -50,159 +63,348 @@ def __init__(self):
 
 class ReadingOrderEditor(QWidget):
+    """
+    A class that represents an editor for a reading order.
+    Combining an ordered list of the current comics in the order
+    on the left and the full collection of comics in the library
+    on the right.
+
+    It is of QWidget type.
+    """
+
     def __init__(self, order_id: int, order_title: str):
```

Ruff (0.15.12)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)